README.md

```diff
@@ -88,92 +112,72 @@ verification component will be mounted. -->
 <div id='onfido-mount'></div>
 ```
 
-### 3. SDK init code
+### 5. Initialising the SDK
 
-An example of how the SDK is initialised with all the available options used. These are broken down into more detail below.
+You are now ready to initialise the SDK:
 
 ```js
 Onfido.init({
-  // the token that you generate on your server
-  token: 'your-jwt-token',
+  // the JWT token that you generated earlier on
+  token: 'YOUR_JWT_TOKEN',
   // id of the element you want to mount the component on
   containerId: 'onfido-mount',
-  // here are various callbacks that fire during the capture process
-  onReady: function() {
-    // callback that fires when successfully authorised
-  },
-  onDocumentCapture: function(capture) {
-    // callback for when the document has captured successfully
-  },
-  onFaceCapture: function(capture) {
-    // callback for when the face capture was successful
-  },
   onComplete: function(capturesHash) {
-    // callback for when everything is complete
-  },
-  steps: [
-    //you can customize the flow of different steps of the SDK flow
-    //you can change the order or remove steps
-    'welcome','document','face','complete'
-  ]
+    console.log('Photos captured');
+  }
 })
 ```
 
-### 4. Change options in runtime
+Congratulations! You have successfully started the flow. Carry on reading the next sections to learn how to:
 
-It's possible to change the options initialised at runtime.
-Example below.
+- Generate JSON Web Tokens (manually or via the Onfido API)
+- Handle callbacks
+- Customise the SDK
+- Upload files to the Onfido API
+- Create checks
 
-```javascript
-onfidoOut = Onfido.init({...})
-...
-//Change the title of the welcome screen
-onfidoOut.setOptions({
-  steps: [
-    {
-      type:'welcome',
-      options:{title:"New title!"}
-    },
-    'document',
-    'face',
-    'complete'
-  ]
-});
-...
-//replace the jwt token
-onfidoOut.setOptions({ token:"new token" });
-...
-//Open the modal
-onfidoOut.setOptions({ isModalOpen:true });
-```
+## JSON Web Token authentication
 
-The new options will be shallowly merged with the previous one. So one can pass only the differences to a get a new flow.
+Clients are authenticated using JSON Web Tokens (JWTs). The tokens are one use only and expire after 30 minutes. See [here](https://jwt.io/) for details on how JWTs work.
 
-## Public options and methods
+You need a new JWT each time you initialise the SDK. You can obtain a JWT in two ways:
 
-A breakdown of the options and methods available to the SDK.
+### 1. Through Onfido API
 
-- **`token {String} required`**
+The Onfido [API](https://documentation.onfido.com/) exposes a JWT endpoint. See the API [documentation](https://documentation.onfido.com/#json-web-tokens) for details.
 
-  A JWT is required in order to authorise with our WebSocket endpoint. If one isn’t present, an exception will be thrown.
+### 2. Generating your own
```

Comment:
Given that we want to deprecate this maybe we should remove it from the documentation